HACKING.md

```diff
@@ -0,0 +1,13 @@
+# Hacking
+
+## Build documentation
+
+Build and open in a Web browser:
+
+```sh
+$ cd doc
+$ make html
+$ xdg-open _build/html/index.html
```

Comment:
I'm not familiar with Sphinx, so I've tried to compare current version with result of this build

This build:
![image](https://user-images.githubusercontent.com/20455996/143588750-c35c369b-a145-4472-9606-62a12343e881.png)

luafun.io:
![image](https://user-images.githubusercontent.com/20455996/143588793-0b9a1e62-7894-4440-b264-338e114e2001.png)

It looks like Sphinx 4.3.0 (with thir PR changes or without) is added an extra newline added before `Contents`. Not sure if it's something worth noting.

Reply:
I'll answer here regarding grey code blocks as well. Your comment:

> It is a matter of personal taste, but I think using grey for code blocks as it was before will be a better idea, if this is easy to set up.

I think it is not hard to do: copy the theme and tweak CSS files. Or, maybe, we can even add our CSS rules without copying the whole theme.

However I don't want step out from 'default haiku theme' (at least within this pull request) mainly because the original documentation uses defaults provided by the theme. A kind of being minimalistic in changes.

Another reason is that anything aside default is a thing of taste and if I'll just declare 'that's default' it should be easier to proceed through review process :)

Reply:
Okay, sound fair